framework/wazuh/tests/test_manager.py

```diff
@@ -39,6 +39,25 @@ def test_manager():
     return test_manager
 
 
+@pytest.mark.parametrize('process_status', [
+    'running',
+    'stopped'
+])
+@patch('wazuh.manager.exists')
+@patch('wazuh.manager.glob')
+def test_status(manager_glob, manager_exists, test_manager, process_status):
```

Comment:
We should add a docstring to explain what the test does.

Reply:
Done here: https://github.com/wazuh/wazuh/pull/2718/commits/b4d6d3b6e8919708414a02f09eff8d463b10850c